src/data/event.js

```diff
@@ -8,6 +8,10 @@ export type Event = {
     startTime: { [string]: string },
     endTime: { [string]: string },
     location: { [string]: { lat: number, lon: number } },
+    addressLine1: { [string]: string },
```

Comment:
If these are optional, can we communicate that here? I think there may be a syntax to do so.

Reply:
👍